packages/backend/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
-  rules: {
-    'no-console': 0, // Permitted in console programs
-    'new-cap': ['error', { capIsNew: false }], // Because Express constructs things e.g. like 'const r = express.Router()'
-  },
+  extends: [
+    require.resolve('@backstage/cli/config/eslint.backend'),
+    '@spotify/eslint-config-oss',
```

Comment:
eslint-config-oss should be picked up by the root config afaik

Reply:
Hm but the root also includes `@backstage/cli/config/eslint` which i DON'T want. I was hoping that it wouldn't 🤔

Reply:
Right so let's keep it and make sure the parent config isn't picked up. Afaik you need to set `root: true` either here or in the shared config.